### DISCUSSION TITLE
Transforming coordinates between two different epochs in the same reference frame (ex: ITRF14@2010.234 to  ex: ITRF14@2012.574)

### DISCUSSION THREAD

**Discussion by paarnes** [pyproj4/pyproj#1164] [Q&A]
What is the best way to perform a transformation between two epochs in the same reference frame (dynamic CRS) in Pyproj? Assume you have a set of coordinates given in ITRF14 with epoch= 2010.234 and want to transform those coordinates to a different epoch like 2012.574, but still in ITRF14. What's the best way to do such a transformation?

The challenge is that the epoch for the input coordinates has to be definened as well. Hence, the same problem also occurs if one tries to transform between two different dynamic reference frames.  I haven't found a good way to do such transformations and hope someone could present a solution.

**Comment by snowman2** [comment]
I recommend asking the PROJ mailing list. More details in the README: https://github.com/osgeo/proj
